src/tests/ftest/datamover/posix_preserve_props.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+from data_mover_test_base import DataMoverTestBase
+from os.path import join
+from pydaos.raw import DaosApiError
+import avocado
+
+class DmvrPreserveProps(DataMoverTestBase):
+    # pylint: disable=too-many-ancestors
+    """Data Mover validation for --preserve-props option only for DFS copies.
+    Test Class Description:
+        Tests the following cases:
+            1. DAOS -> POSIX: Write container properties to an hdf5 file, and write
+               to destination posix path.
+            2. POSIX -> DAOS: Read container properties from an hdf5 file, and
+               write to destination container.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super().setUp()
+
+        # Get the parameters
+        self.ior_flags = self.params.get(
+            "ior_flags", "/run/ior/*")
```

Comment:
This would probably fit on one line now we have 100 character lines, and is much easier to read.
```suggestion
        self.ior_flags = self.params.get("ior_flags", "/run/ior/*")
```